packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/waiter-state-machine.ts

```diff
@@ -0,0 +1,82 @@
+// The contents of this file were adapted from the AWS CDK provider framework.
+// https://github.com/aws/aws-cdk/blob/c52ff08cfd1515d35feb93bcba34a3231a94985c/packages/aws-cdk-lib/custom-resources/lib/provider-framework/waiter-state-machine.ts
```

Comment:
Modifications:
```diff
-        'framework-onTimeout-task': {
-         End: true,
-          Type: 'Task',
-          Resource: props.timeoutHandler.functionArn,
-        },
```

We're depending on the CloudFormation custom resource default (and unchangeable) timeout of 1 hour. We no longer need a timeout function.

Reply:
Would it simplify our future maintenance (by making it easier to compare changes from the CDK) to leave this unchanged, and simply add a noop timeout handler and default timeout value high enough to never be invoked?

Reply:
Agreed that it would likely help with maintainability, but that comes at the cost of the "fake" timeout value and timeout task state (`framework-onTimeout-task`) being potentially misinterpreted by future readers as "real." 

I don't have a strong opinion either way. If you think it's worthwhile to add back, happy to do it.